apps/spruce/cypress/integration/image/event_log.ts

```diff
@@ -0,0 +1,24 @@
+describe("event log page", () => {
+  const IMAGE_EVENT_LIMIT = 5;
+  it("load more button should return twice as many events", () => {
+    cy.visit("/image/ubuntu2204/event-log");
+    cy.dataCy("image-event-log-card").should("have.length", IMAGE_EVENT_LIMIT);
+    cy.dataCy("load-more-button").click();
+    cy.dataCy("image-event-log-card").should(
+      "have.length",
+      2 * IMAGE_EVENT_LIMIT,
+    );
+  });
+
+  it("should show no events when non-existent filter is applied", () => {
```

Comment:
nit. `nonexistent filter` sounds a bit confusing (the filter does exist, but the item does not)
```suggestion
  it("should show no events when filter for a nonexistent item is applied", () => {
```